src/styles/_variables.scss

```diff
@@ -17,26 +20,24 @@ $color-button-hovered: #005d92;
 // Light:
 $color-text-light: rgba(black, 0.85);
 $color-text-light-alt: rgba(white, 0.85);
-$color-bg-light-1: #ccd1d6;
-$color-bg-light-2: #e6e8eb;
+$color-accent-purple: #a000bc;
+$color-bg-light-1: #ffffff;
+$color-bg-light-2: #f8f9fa;
 $color-bg-light-3: #f0f1f3;
-$color-bg-light-4: #f8f9fa;
-$color-bg-light-5: #eeeeee;
+$color-bg-light-4: #e6e8eb;
+$color-bg-light-5: #dadee2;
+$color-bg-dev-light-1: #ccd1d6;
```

Comment:
It would be worth adding a note to explain where this variable comes from, and what the `dev` indication here means - I presume this is a new color specified here that was used only in the codebase but not specified in the styleguide?